ui/scripts/package.sh

```diff
@@ -31,3 +32,9 @@ cd $ROOT_DIR/dist && zip -r ../target/${ZIP_FILE_NAME}_chrome.zip *
 cd ..
 grunt android
 cd $ROOT_DIR/dist && zip -r ../target/${ZIP_FILE_NAME}_android.zip *
+
+if [ -n $XVFB_PID ]; then
+    echo "Killing Xvfb process $XVFB_PID"
+    kill $XVFB_PID
```

Comment:
We also need to make sure we kill the Xvfb process if we started it--otherwise this script does not exit.